util/logging.h

```diff
@@ -11,40 +11,47 @@
 // with macros.
 
 #pragma once
-#include "port/port.h"
 
 // Helper macros that include information about file name and line number
-#define STRINGIFY(x) #x
-#define TOSTRING(x) STRINGIFY(x)
-#define PREPEND_FILE_LINE(FMT) ("[" __FILE__ ":" TOSTRING(__LINE__) "] " FMT)
+#define ROCKS_LOG_STRINGIFY(x) #x
+#define ROCKS_LOG_TOSTRING(x) ROCKS_LOG_STRINGIFY(x)
+#define ROCKS_LOG_PREPEND_FILE_LINE(FMT) ("[%s:" ROCKS_LOG_TOSTRING(__LINE__) "] " FMT)
+
+inline const char* ROCKS_LOG_FILENAME(const char* file)
```

Comment:
This is a function, so naming is `RocksLogFileName()`.

Reply:
My intention is to make it look like a macro instead of a function.
Because it doesn't make sense to call it anywhere else.

But I am okay to name it as a function.

Reply:
Done.
Change the name to `RocksLogShorterFileName`.
Thanks.